packages/internal/package.json

```diff
@@ -16,6 +16,10 @@
   "files": [
     "dist"
   ],
+  "exports": {
+    ".": "./dist/index.js",
+    "./routes": "./dist/routes.js"
```

Comment:
Allows us to import like this:
```
    const { getProjectRoutes } = await import('@redwoodjs/internal/routes')

```

We'll need it in the next stage, I just broke up the PR